workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGen/EntryPoint.cs

```diff
@@ -131,7 +144,7 @@ private bool ValidateOptions()
 
             if (string.IsNullOrEmpty(options.SchemaCompilerPath))
             {
-                Console.WriteLine("Schema compiler location not specitied");
+                Console.WriteLine("Schema compiler location not specified");
```

Comment:
Wow this must have been here for a _long_ time. This just never gets hit in the normal flow